gateway-ha/src/main/java/io/trino/gateway/ha/persistence/JdbcConnectionManager.java

```diff
@@ -105,5 +106,23 @@ private void startCleanUps()
                 1,
                 120,
                 TimeUnit.MINUTES);
+
+        executorService.scheduleWithFixedDelay(
```

Comment:
Follow up task for later maybe .. we need a feature in the UI to wipe stale cookies maybe?

Reply:
i'll open an issue once this is in